### PR TITLE
[FIX #34] SIze of  social media links

### LINKED ISSUE #34: Size of Social Media Links
I feel the size for the social media links are  very small and should be bigger than their present size.  

Because links to share must be highlighted.

### PR DESCRIPTION
**Pull request by amoghj8**
The size of social media links have been altered.

Link: https://amoghj8.github.io/inpycon2017/

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
@amoghj8 when you are making changes for front-end task. And you can serve your site on gh-pages. Always include a link (gh-pages live link) in PR to view those changes. 
Although i am not a reviewer but surely it makes task of reviewers easy.

**Comment by amoghj8** [comment]
Thanks for the info @pradeepgangwar , I haven't done that before. Should I add gh-page to my forked repo?

**Comment by pradeepgangwar** [comment]
@amoghj8 Yes, and paste a link here for reviewers to see it :+1:

File: css/main.css [hunk 1]
```diff
@@ -1165,6 +1165,19 @@ li.tg-rss a{background:#ff6600;}
 	width:38px;
 	line-height:38px;
 }
+
+@media screen and (max-width: 767px) {
```
Comment (CuriousLearner) [inline css/main.css]: This `max-width` 767 px and `min-width` 768 px is really odd.

Can we improve it?

**Review by CuriousLearner — CHANGES_REQUESTED** [review]
I've left my review in-line.

Thanks for your contributions @amoghj8 !

File: css/main.css (hunk 1 above)
Comment (amoghj8) [reply css/main.css]: @CuriousLearner  How about "Responsive utilities" of bootstrap? We can edit the sizes using visible or hidden classes.

Comment (SaptakS) [reply css/main.css]: @amoghj8 the thing what @CuriousLearner is trying to say I believe is you can set either one (wither max-width or min-width) and set the other as the default font-size. So something like by default it is 2em, but if media screen width below 767px (i.e max-width:767px) then `font-size:1.5em`. Keeping both max-width and min-width is weird.

Comment (amoghj8) [reply css/main.css]: @SaptakS  Oh ! That's true. Sorry for the silly mistake. Shall I edit and send a PR?

Comment (sayanchowdhury) [reply css/main.css]: Just update the PR

**Comment by CuriousLearner** [comment]
Hey @amoghj8 !

Can you please resolve the conflicts so that we can review and get this merged in?

**Comment by amoghj8** [comment]
@CuriousLearner  Resolved the conflicts

**Comment by CuriousLearner** [comment]
@saptakS Reviews?

**Review by SaptakS — APPROVED** [review]

**Review by CuriousLearner — APPROVED** [review]
LGTM!

Thanks for your contributions! Merging in!

**Pull request merged**